api.swagger.yaml

```diff
@@ -6369,6 +6446,183 @@ components:
           format: uuid
           example: 01234567-89ab-cdef-1234-56789abcdef0
       type: object
+    Activity:
+      required:
+        - user
+        - entity
+        - timestamp
+        - likes
+        - comments
+        - link
+        - short_text
+        - topic
+      properties:
+        timestamp:
+          description: This is the timestamp (in seconds) of when the activity was created.
+          type: integer
+          example: 1510941596
+        short_text:
+          description: >-
+            Contextual text for the activity, with placeholders @user for the user's name and optionally @group for the group name.
+          type: string
+          example: "@user created the following within @group"
+        topic:
+          description: 'This is topic of the feed, used for filtering'
+          type: string
+          enum:
+            - people
+            - groups
+            - content
+        user:
+          type: object
+          required:
+            - uuid
+            - user_avatar
+            - display_name
+          description: The user who triggered the activity.
+          properties:
+            uuid:
+              description: The UUID of the user.
+              type: string
+              format: uuid
+              example: 01234567-89ab-cdef-1234-56789abcdef0
+            image:
+              type: object
+              description: Avatar image of the user.
+              properties:
+                url:
+                  type: string
+                  format: uri
+                  example: >-
+                    https://dev.invotra.com/system/files/user-picture/large_avatar/1f0c6cb5-a3c3-4197-bebc-b3e44462dc24/profile.png
+            username:
+              description: The username of the user.
+              type: string
+              example: Dorothy.Whitehead@akagera.com
+            display_name:
+              description: The display name of the user.
+              type: string
+              example: Dorothy Whitehead
+            teams:
```

Comment:
Why do we need teams/locations/job roles?

Reply:
Haven't a clue - they were added in #97, I just copied them here.

Reply:
Managed to dig it out, it's for the 'xxx updated their profile' activities: https://projects.invisionapp.com/share/NCHTCADAEWU#/screens/294512296_Feed-Home